src/components/ShowContextMenuContext.js

```diff
@@ -32,7 +32,8 @@ function showContextMenuForReport(event, anchor, reportID, action, checkIfContex
         '',
         anchor,
         reportID,
-        action,
+        action.reportActionID,
+        reportID,
```

Comment:
Is it necessary to pass `reportID` twice? I see it a couple lines above too.

Reply:
Oh, I see that maybe it is OK as in another place you are passing `originalReportID` here. Could you maybe add code comments to each place to explain why it's different (or the same) as `reportID`?